lib/commands/TheoCommand.js

```diff
@@ -0,0 +1,16 @@
+import Command from './Command'
+
+const fs = require('fs')
+
+const randTheo = ({ message }) => {
+    const quotes = fs.readFileSync('data/theo.txt').toString().split('\n')
```

Comment:
Let's pull this out into the top-level so that we don't do a file access every time this command is invoked.